src/PerfView/GuiUtilities/WindowBase.cs

```diff
@@ -15,8 +15,17 @@ public class WindowBase : Window
 
         public WindowBase(Window parentWindow)
         {
-            Owner = parentWindow;
-            WindowStartupLocation = WindowStartupLocation.CenterOwner;
+            if (parentWindow != null)
+            {
+                // Try to set the owner, but it might fail (if the window has never been displayed)
```

Comment:
❔ Is `window` here referring specifically to `parentWindow`? If so, it may help to do this:

```suggestion
                // Try to set the owner, but it might fail (e.g. if parentWindow has never been displayed)
```